.circleci/config.yml

```diff
@@ -39,9 +39,9 @@ jobs:
     - restore_cache:
         key: bin-cache
     # Run autogeneration and see if there are differences
-    # TODO: Fix this; somehow the docs/api/ignite_v1alpha1.md ends up being malformatted in CI; it adds
+    # TODO: Fix this; somehow the docs/api/ignite_v1alpha2.md ends up being malformatted in CI; it adds
```

Comment:
Could you verify if this workaround is still necessary?

Reply:
Removed it and looks like we no longer need it 😃 
Thanks.